src/libxtp/qmpackage.cc

```diff
@@ -140,5 +138,118 @@ std::string QMPackage::FindDefaultsFile() const {
   return xmlFile;
 }
 
+void QMPackage::ReorderMOs(Eigen::MatrixXd& v,
```

Comment:
The methods from here to end of the file are some much better and cleaner! Awesome work :+1: